ConsoleApp3/ConsoleApp3/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите число x");
+            double x = double.Parse(Console.ReadLine());
+            if (x < -2)
+            {
+                Console.WriteLine(Equation1(x));
+            }
+            if (x > 1)
+            {
+                Console.WriteLine(Equation3(x));
+            }
+            else
+            {
+                Console.WriteLine(Equation2(x));
+            }
```

Comment:
Лучше все записать одним оператором. И скобки в данном случае (после `if` один оператор) можно не ставит.
```suggestion
            if (x < -2)
                Console.WriteLine(Equation1(x));
            else if (x > 1)
                Console.WriteLine(Equation3(x));
            else
                Console.WriteLine(Equation2(x));
```